src/commands/up.rs

```diff
@@ -113,6 +114,11 @@ pub struct UpCommand {
     #[clap(long, takes_value = false, env = ALWAYS_BUILD_ENV)]
     pub build: bool,
 
+    /// Specific component to run. Can specify multiple. If omitted, all
+    /// components are run.
+    #[clap(hide = true, long = "component")]
```

Comment:
Is `hide=true` here because this is experimental?

Reply:
Yeah. I am not sure if there is another way we've marked features as experimental in the past. Should i update the comment to say it is experimental too?

Reply:
yea not sure what the precedent is but I think a comment would be helpful.